_sass/components/_header.scss

```diff
@@ -0,0 +1,72 @@
+.c-hero {
+  width: 100%;
+  height: 100%;
+  background: url('../assets/logo-background.svg'), $top-gradient, url("../assets/homepage-clouds.jpg");
+  background-repeat: no-repeat;
+  background-size: 800px, cover, 200%;
```

Comment:
Would it be possible to change any instance of `px` that is larger than `1px` to `rem`?
You can do that by dividing the `px` value by 16. So this one would be `800/16 = 50`